### DISCUSSION TITLE
RaspBee II e deCONZ

### DISCUSSION THREAD

**Discussion by morphy81** [indomus/forum#920] [ZigBee, Z-Wave, Thread, Matter]
Salve a tutti, ho seguito la guida per installare il Raspbee II e deCONZ su HomeAssistan OS, ora peò sulla barra laterale quando premo su deCOnz mi compare una doppia scelta "Phoscon" o "deCONZ" se premo il primo entro nell'interfaccia che è indicata nella guida ma non riesco in nessun modo ad intregrare i sensori porte e finestre Aqara ne una lampada Philips Hue (che con i rispettivi hub funzionano). Se prevo invece il secondo pulsante mi porta in un interfaccia tipo finestra di Windows in cui mi dice di connettermi al device ed è indicato "Raspbee" se premo però sta un attimo e poi torna la scritta connect da premere e nno cambia nulla.
Che posso fare per risolvere? Ho verificato nelle impostazione di HA che il dispositivo Raspbee II è presente col nome ttyAMA0 quindi dovrebbe essere correttamente installato.

Grazie a tutti!

**Comment by indomus** [comment]
> Salve a tutti, ho seguito la guida per installare il Raspbee II e deCONZ su HomeAssistan OS, ora peò sulla barra laterale quando premo su deCOnz mi compare una doppia scelta "Phoscon" o "deCONZ" se premo il primo entro nell'interfaccia che è indicata nella guida ma non riesco in nessun modo ad intregrare i sensori porte e finestre Aqara ne una lampada Philips Hue (che con i rispettivi hub funzionano). Se prevo invece il secondo pulsante mi porta in un interfaccia tipo finestra di Windows in cui mi dice di connettermi al device ed è indicato "Raspbee" se premo però sta un attimo e poi torna la scritta connect da premere e nno cambia nulla. Che posso fare per risolvere? Ho verificato nelle impostazione di HA che il dispositivo Raspbee II è presente col nome ttyAMA0 quindi dovrebbe essere correttamente installato.
> 
> Grazie a tutti!

Ci fai uno screenshot per capire meglio?
P.S. non ti conviene usare direttamente ZigBee2MQTT?
